syntax/vaffle.vim

```diff
@@ -2,16 +2,17 @@ if exists('b:current_syntax')
   finish
 endif
 
-
 syn match vaffleDirectory '^  .\+/$'
 syn match vaffleHidden '^  \..\+$'
+syn match vaffleHiddenWithIcon '^  . \..\+$'
 syn match vaffleSelected '^* .\+$'
 syn match vaffleNoItems '^  (no items)$'
 
+
 hi! def link vaffleDirectory Directory
 hi! def link vaffleHidden Comment
+hi! def link vaffleIconRenderHidden Comment
```

Comment:
Ah, I missed it! Please rename `vaffleIconRenderHidden` to new one.

Reply:
oh, I'm sorry. I forget to fix.